featuretools/primitives/utils.py

```diff
@@ -59,7 +59,20 @@ def get_transform_primitives():
                           featuretools.primitives.TransformPrimitive):
                 if attribute.name:
                     transform_primitives.add(attribute)
-    return list(transform_primitives)
+    return {prim.name: prim for prim in transform_primitives}
+
+
+def list_primitives():
+    transform_primitives = get_transform_primitives()
+    agg_primitives = get_aggregation_primitives()
+    transform_df = pd.DataFrame({'name': list(transform_primitives.keys()),
+                                 'description': [prim.__doc__.split("\n")[0] for prim in transform_primitives.values()]})
```

Comment:
This gives different results between docstrings that are
```
"""
Words go here
"""
```
``` 
"""Words go here
"""
```
and 
```
""" Words go here
"""
```
We have all three but should probably settle on one.

Reply:
we use the Google style docstrings for all public functions. for some private functions, we may be inconsistent and should fix those as we see it. 

here's the Google style docstrings guide: http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
I've pushed a version which has nonempty first line for all of the primitives. It looks like there's some extra space being added by something in the description (as shown in this image)

![image](https://user-images.githubusercontent.com/27442755/38385089-f34a9170-38de-11e8-84f6-b046774c681d.png)